src/components/Cell/Cell.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './Cell.css'
+import { useDispatch } from 'react-redux'
+import { click } from 'modules/game'
+
+
+export const Cell = (() => {
+
+    const dispatch = useDispatch()
+    const onClick = () => {
+        dispatch(click())
+    }
+    return (
+        <div className='cell' onClick={onClick}></div>
+    )
```

Comment:
```suggestion
    const onClick = () => {
        dispatch(click())
    }
    
    return (
        <div className='cell' onClick={onClick}></div>
    )
```
Par convention et pour aérer ton code, il faut toujours un saut de ligne avant un return